PositionalGuide/Plugin.cs

```diff
@@ -23,8 +23,15 @@ namespace PrincessRTFM.PositionalGuide;
 public class Plugin: IDalamudPlugin {
 	public const string Command = "/posguide";
 
-	public const double ArcLength = 360 / 16;
-	public const int ArcSegmentCount = 8;
+	private const int circleSegmentCount = 128;
+	// lineIndexToRad and circleSegmentIdxToRad are fixed and only need to be calculated once at the start,
+	// since neither the number of lines nor the number of circle segments change
+	private readonly float[] lineIndexToAngle = (from idx in Enumerable.Range(Configuration.IndexFront, Configuration.IndexFrontLeft + 1)
+											   select (float)(idx * Math.PI / 4)).ToArray();
+	private readonly float[] circleSegmentIdxToAngle = (from idx in Enumerable.Range(0, circleSegmentCount)
```

Comment:
This and the above line use LINQ with query expression syntax. How do you feel about changing it to use method call syntax instead? It's mostly a personal preference, but my code has always used method syntax, and it would prevent mixed indentation like is present here.